### PR TITLE
Name fixes and showcard

### PR DESCRIPTION
**Pull request by Nderkovich**

### CODE REVIEW

File: sass/footer.sass [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    content: "\e901"
+
+.ok::after
+    content: "\e900"
+
+.insta::after
+    content: "\e903"
+
+@media screen and (min-width: 900px) and (max-width: 1280px)
+    .d
```
Comment (uladzislau-kirychenka) [inline sass/footer.sass]: what does it mean?

File: sass/footer.sass [hunk 2]
```diff
@@ -0,0 +1,72 @@
+.footer
+    max-width: 100%
+    margin: auto
+    display: flex
+    height: 160px
+    background-color: #ffffff
+    flex-direction: row
+    justify-content: center
+
+    &__left,
```
Comment (uladzislau-kirychenka) [inline sass/footer.sass]: use more suitable naming

File: sass/header.sass [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+                color: #0d3351
+
+            &_currency::before
+                content: "\e906"
+                font-family: "icomoon"
+                font-size: 13px
+                margin-right: 5px
+                color: #33c7fd
+
+    &__phone-descr
```
Comment (uladzislau-kirychenka) [inline sass/header.sass]: please don't reduce names

File: sass/header.sass [hunk 4]
```diff
@@ -150,19 +150,19 @@
         &__bar
             display: none
 
-        &__content
-            width: 70%
-            margin: auto
-            padding-top: 20px
+    .header__content
```
Comment (uladzislau-kirychenka) [inline sass/header.sass]: why you have removed `&`?

Comment (Nderkovich) [reply sass/header.sass]: I thought that previous selector was already big enough, and tripple nesting with this selector wouldnt look clean

**Pull request merged**